src/playlist-maker.js

```diff
@@ -1,8 +1,16 @@
 import window from 'global/window';
-import assign from 'object.assign';
 import playItem from './play-item';
 import * as autoadvance from './auto-advance';
 
+// Lightweight Object.assign alternative.
+const assign = (target, source) => {
+  for (let key in source) {
+    if (source.hasOwnProperty(key)) {
+      target[key] = source[key];
+    }
+  }
+};
```

Comment:
The `object.assign` polyfill was adding quite a bit of heft to the project. More than it was worth for only being used in one place.